src/machine/machined.c

```diff
@@ -183,9 +229,13 @@ static int manager_connect_bus(Manager *m) {
         assert(m);
         assert(!m->bus);
 
-        r = sd_bus_default_system(&m->bus);
+        if (arg_user)
+                r = sd_bus_default_user(&m->bus);
+        else
+                r = sd_bus_default_system(&m->bus);
+
```

Comment:
doesn't matter much, but we try to keep function calls and their immediate error handling together, i.e. not spurious empty lines in between.

Reply:
ok